src/inputs/checkbox.jsx

```diff
@@ -1,13 +1,22 @@
-import React, { Component } from 'react';
-import _ from 'lodash';
+import {
+    isFunction,
+} from 'lodash';
+import React from 'react';
 import ClassNames from 'classnames';
 import PropTypes from 'prop-types';
 import Icon from '../dataDisplay/icon';
 
 const propTypes = {
-    align: PropTypes.oneOf([ 'left', 'right' ]),
+    align: PropTypes.oneOf(['left', 'right']),
     checked: PropTypes.bool,
     className: PropTypes.string,
+    /**
+     * A TableRow can be disabled.
```

Comment:
Oh no ... copy and paste here.  Should be "A Checkbox can be disabled."

Reply:
Fixed

Reply:
Yep, confirmed it looks good as of c17198b4.
Resolving.